src/device-state.ts

```diff
@@ -319,6 +320,8 @@ export async function setTarget(target: TargetState, localSource?: boolean) {
 	// This will throw if target state is invalid
 	target = parseTargetState(target);
 
+	await writeAndSyncFile('/tmp/balena-supervisor/target-state', target);
```

Comment:
@jaomaloy I was under the impression the agreement was to dump only the device configuration section, not the whole target state, as that's the part the OS might be interested. Maybe listen to the arch call recording to confirm, or @pipex can chip in.

Reply:
Yes, that was the agreement. You'll need to get that from the local target (see below line 335). The config property has all the configuration variables. 

You'll also need to stringify the value (the variable contains an object), in order to call `writeAndSync` as the error above suggests

Reply:
Yes, let's just dump the config.